_opensearch/rest-api/nodes-apis/nodes-reload-secure.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Nodes reload secure settings
+parent: Nodes APIs
+grand_parent: REST API reference
+nav_order: 50
+---
+
+# Nodes reload secure settings
+
+The nodes reload secure settings endpoint allows you to change secure settings on a node and reload the secure settings without restarting the node.
+
+## Path and HTTP methods
+
+```
+POST _nodes/reload_secure_settings
+POST _nodes/<node id>/reload_secure_settings
+```
+
+## Path parameter
+
+You can include the following optional path parameter in your request:
```

Comment:
(suggestion): if you only have one parameter, you can remove the table and just define it like this:
"You can specify individual node names you want to reload. Provide the `node_id` (type: String) in your request.

Reply:
I think it will be good to leave it as a table for visual consistency between pages.